src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -104,6 +107,10 @@ private static void aggregateBlameAuthorModifiedAndDateInfo(RepoConfiguration co
             }
 
             if (config.isLastModifiedDateIncluded()) {
+                if (config.isShallowCloningPerformed()) {
```

Comment:
If the two options already mutually exclusive by the argparser, will it still be possible to go to the branch here?

Reply:
It is possible, since the shallow cloning can also be enabled as an option in the repo-config.csv. So as @Tejas2805 had mentioned above, it is possible for a user to specify 1 repo out of 5 to be shallow cloned while also using the `--last-modified-date` flag.